css/openmicroscopy.css

```diff
@@ -275,6 +275,7 @@ h3.subdivider {
     font-weight: 600;
     margin-bottom: 5px;
     text-shadow: 1px 1px 3px rgba(0,0,0,1);
+    line-height: 1rem;
```

Comment:
I think this addition has broken the line spacing on all pages, and not just https://github.com/ome/www.openmicroscopy.org/pull/303#issuecomment-516770392

https://snoopycrimecop.github.io/www.openmicroscopy.org/teams/
<img width="705" alt="Screen Shot 2019-08-01 at 10 55 05" src="https://user-images.githubusercontent.com/1644105/62284107-e7bccb00-b44a-11e9-8220-20361d146268.png">

https://www.openmicroscopy.org/teams/
<img width="650" alt="Screen Shot 2019-08-01 at 10 55 10" src="https://user-images.githubusercontent.com/1644105/62284114-ed1a1580-b44a-11e9-8a3e-d8e7013554c2.png">

@jburel If this is the problem then https://github.com/ome/www.openmicroscopy.org/pull/303/commits/b1038151a5ed018b85a4cf53b97b9b390cf0fef7 can be reverted